docs/library/lcd160cr.rst

```diff
@@ -341,7 +343,8 @@ Advanced commands
 
 .. method:: LCD160CR.jpeg(buf)
 
-    Display a JPEG.  `buf` should contain the entire JPEG data.
+    Display a JPEG.  `buf` should contain the entire JPEG data. JPEG data should
```

Comment:
Maybe we can add a note about the supported encoding.
`Baseline DCT, Huffman coding, 8 bits per sample, 3 color components, YCbCr4:2:2`